app/controllers/api/v1/flights_controller.rb

```diff
@@ -0,0 +1,109 @@
+class Api::V1::FlightsController < ApplicationController
+  protect_from_forgery with: :null_session
+  skip_before_action :verify_authenticity_token, only: %i[create destroy]
+  before_action :set_flight, only: %i[show edit update destroy]
+
+  def index
+    @flights = Flight.all.order(created_at: :desc)
+    @user = current_user
+
+    response_data = {
+      user: @user.as_json(only: %i[name email id]),
+      flights: @flights.as_json(except: %i[created_at updated_at]).map do |flight|
+        {
+          name: flight['name'],
+          picture: flight['picture'],
+          reserved: flight['reserved'],
+          userId: flight['user_id'],
+          basePrice: flight['base_price'],
+          availableSlots: flight['available_slots'],
+          id: flight['id']
+        }
+      end
+    }
+
+    render json: response_data, status: :ok
```

Comment:
Missing error handling here, what happens if it does not work the way you expect it to?